BrainPortal/app/views/users/new.html.erb

```diff
@@ -76,6 +76,8 @@
     <p><label for="no_password_reset_needed">No need to reset initial password:</label>
     <%= check_box_tag :no_password_reset_needed, "1", params[:no_password_reset_needed] == "1" %>
 
+    <%= hidden_field_tag :signup_id, params[:signup_id] %>
```

Comment:
What is this used for?!? There is no code in users_controller that reads that value.

On the other hand, it SHOULD be used in the `create` action, but the cold there has not been adjusted:

```ruby
# Line 144 of users_controller
if signup = Signup.where(:login => @user.login, blahbah)
```